config/src/config/storage_config.rs

```diff
@@ -94,6 +94,7 @@ pub const NO_OP_STORAGE_PRUNER_CONFIG: StoragePrunerConfig = StoragePrunerConfig
     ledger_prune_window: 0,
     ledger_pruning_batch_size: 10_000,
     state_store_pruning_batch_size: 10_000,
+    user_pruning_window_offset: 2000,
```

Comment:
and 2000 is half a second  by our performance standard..

Reply:
`NO_OP_STORAGE_PRUNER_CONFIG` => Pruner is turned off?  I don't see the concern, or I guess I can set it to 0.

Reply:
HAHAHA, that's true, but why request change, so harsh.

Reply:
🗡️ that's for the offset discussion (and "I disagree with David")